templates/problem/voting-stats.html

```diff
@@ -0,0 +1,146 @@
+<style>
+    .vote-stats-background{
+        background-color: rgb(255,255,255);
+        padding: 20px;
+        border-radius: 25px;
+        justify-content: center;
+        align-items: center;
+        margin: auto;
+        text-align: center;
+    }
+
+    .canvas{
+        border: 1px;
+        border: solid;
+        border: #000000;
+    }
+
+    .vote-stats-value{
+        font-size: 24px;
+        font-family: "Segoe UI", "Lucida Grande", Arial, sans-serif;
+    }
+
+    .vote-stats-info{
+        font-size: 24px;
```

Comment:
Any reason why we're making the font so much larger than the rest of the font on DMOJ?

Reply:
What is the standard font? I think this was just done for visual purposes but we could revert.

Reply:
Turns out the fonts also affect how big the graph is.